### DISCUSSION TITLE
Tuning 70 heroics

### DISCUSSION THREAD

**Discussion by pablitoF** [chromiecraft/chromiecraft#4934] [Suggestions]
From my experience 70 untuned heroics on wotlk are too easy compared to 70 heroics on a TBC client where heroics actually proved to be quite the challenge.

I wish for heroics to be tuned because it makes party members commit to teamwork and it makes dungeons more satisfying to complete instead of it being a boring routine cake walk where bosses and mobs get taken down in a few seconds.

**Comment by balleny** [comment]
a first step for them to be "tuned" naturally, is if the trash mobs have their abilities.
from the 5 dungeons available currently, we already saw plenty of trash mobs without abilities.

If you want to improve on that in the first place, have a go on the PTR and create respective bug reports. If the trash mobs have their abilities corrected, heroics will be a lot more difficult. (esp. if you compare to TBC, heroics were difficult because of the trash - not because of the bosses)

a potentially tuning would be nice - although not in the extend it was during ubrs times. Still, before the tuning, we should have a better base line as it might be right now.
